### ISSUE TITLE
Buff the Recipe of Bio SoC

### ISSUE THREAD

**Issue by JustAlkaid** [GTNewHorizons/GT-New-Horizons-Modpack#18563]
### Your GTNH Discord Username

Alkaid

### Your Pack Version

2.6.1

### Your Proposal

Buff the recipe of Bio SoC. Here are 3 ideas (numbers need more discussion):

1. Multiply the recipe output of Living Bio Chip by 16. 
2. Replace the Hypogen dust in the recipe of Living Bio Chip with Transcendent Metal dust, and multiply the output by 8.
3. Remove the Hypogen dust in the recipe of Living Bio Chip, and replace 8000L Sterilized Bio Catalyst Medium
with 9L Molten Proto-Halkonite Steel Base.

### Your Goal

In general, players get a new type of SoC every tier from IV, which means they only need to keep 1 mainly used circuit production line and cover other lower tier circuit demand with SoCs. 
However, as the SoC unlocked in UEV, Bio SoC is reported to be incomprehensibly expensive by almost every player I have met. Therefore, they always kept their Wetware line for all circuits from LuV to UHV until they obtained Transdimensional Alignment Matrix.
Since the original recipe uses Hypogen, I assume that Bio SoC can be used immediately after the DTPF is completed, with a small demand of DTPF material. As a result, I suggest the 3 buffed recipes above. All of them require DTPF to craft, which means the buffed recipe doesn't change the tier of the recipe, but they are cheaper and more economical for late UEV players. So they can tear down their Wetware line and embrace cheap ZPM circuits in a single step.

### Your Vision

Make Bio SoC economical in Late UEV and UIV.

### Final Checklist

- [X] I have searched this issue tracker and there is nothing similar already. Posting on a closed issue saying *I like this change please reconsider adding it* will prompt us to investigate and reopen it once we confirm your report.
- [X] I understand this change request may not attract enough attention and thus not be implemented.
- [X] I understand this change request may be rejected due to other community members think it's inappropriate.
- [X] I believe this feature would make the pack better.

**Comment by Ruling-0** [comment]
I personally found the SoC recipe affordable enough even before convergence for making mainframes, and you have to consider that buffing the recipe has implications far outside of UEV, given bios are presently one of two options viable for mass endgame circuits (UEVs/UHVs are needed in significant quantities).

**Comment by StaffiX** [comment]
I think the speed of the SoC is far more important than the cost, and since SoCs usually only becomes worthwhile to switch after around 2 overclocks so even after making the recipe cheaper, people would only switch at around UMV.

Plus the infra required for SoC is also quite big still, like the autoclave step and chromatic glass bolt making.

**Comment by JustAlkaid** [comment]
> I personally found the SoC recipe affordable enough even before convergence for making mainframes, and you have to consider that buffing the recipe has implications far outside of UEV, given bios are presently one of two options viable for mass endgame circuits (UEVs/UHVs are needed in significant quantities).

Yeah this is exactly what I wanted. The buff is to unburden players from having to build machines in large numbers due to using old recipes. For example, because players had to scale LCRs for platline, naqline and so on, we now have QFTs which enable players to skip them  in later stages. This principle also applies to SoCs - players are encouraged to make them and use them as soon as possible. So that SoCs should be affordable(use less high-tier materials to replace amounts of low-tier materials) and have its advantage(to skip some steps, to craft circuits in 1 step).

About your concern I asked several players in UEV who had no Matrix. All of them reported that Hypogen was the biggest obstacle that prevented them from using the SoC recipe. Hypogen abuse can also be seen in other recipes like mk4 naqfuel. The old naqfuel mk4 recipe uses 1440L Hypogen for 500L fue, and the recipe was eventually buffed in 2.7.0. In my opinion, players always use the first batch of Hypogen to make coils, then spacetime. Use such a time-consuming material to make a small amount of circuits seemed unwise.

However, the effect of this may-be-existed buff for endgame do needs more discussion. It is important to find a balance between lower stage and higher stage.

**Comment by JustAlkaid** [comment]
> I think the speed of the SoC is far more important than the cost, and since SoCs usually only becomes worthwhile to switch after around 2 overclocks so even after making the recipe cheaper, people would only switch at around UMV.
> 
> Plus the infra required for SoC is also quite big still, like the autoclave step and chromatic glass bolt making.

The time is a problem but I don't think it should be changed. There are players using Crystal Mainframe as theri UV circuits in UMV for speed. If they want to use more complicated recipe for speed, it is their choices. I have not calculated but if Bio SoC can be used widely, the AAL crafting Bio Processing Units can be saved - the AAL have no perfect overclock!

About the autoclave step and the bolt, I asked some players. They said that the autoclave multi in 2.7.0 is unbelievably fast(350% speed bonus for Electrum Flux Coil Block, 72 parallels for Quantium Pipe block, both of them can be crafted easily in UV) so it was not a problem I think. The chromatic glass is also used in naqfuel at a high rate, and it is much more quicker than Hypogen in DTPF so players can directly use the prepared chromatic glass for naqfuel(or optical circuits).
